src/main/java/uk/co/onsdigital/discovery/dao/DatasetDAOImpl.java

```diff
@@ -70,16 +88,16 @@ private static String getStr(ResultSet rs, String key) throws SQLException {
 
 
     // Function encapsulates the creation of the MapSqlParameterSource. Can be easily be swapped for mock in test.
-    private Function<NamedParam[], MapSqlParameterSource> createParameterSource = (tuples) -> {
+    private Function<List<NamedParam>, MapSqlParameterSource> createParameterSource = (list) -> {
         MapSqlParameterSource sqlParameterSource = new MapSqlParameterSource();
-        for (NamedParam namedParam : tuples) {
+        for (NamedParam namedParam : list) {
             sqlParameterSource.addValue(namedParam.getKey(), namedParam.getValue());
         }
         return sqlParameterSource;
     };
 
     @Override
-    public List<String> getDatasetIds() throws MetadataEditorException {
+    public java.util.List getDatasetIds() throws MetadataEditorException {
```

Comment:
Yeah dunno what has happened there. Might have been a refactoring error. Ill change it back